src/simplexgrid.jl

```diff
@@ -44,6 +44,19 @@ function simplexgrid(coord::Array{Tc,2},
     return grid
 end
 
+function simplexgrid(coord::Array{Tc,2},
```

Comment:
Who shall calculate the bedgeregions here ? IMHO these shall come from TetGen in this call, and
be accompanied by a bedgenodes array. 

Tetgen has input and output of boundary edges with markers, we however never really tried this...
https://github.com/JuliaGeometry/TetGen.jl/blob/master/src/rawtetgenio.jl#L545

Reply:
I was first thinking about tensor grids. I would propose to have an additional flag for the related simplexgrid constructors to also compute the bedgenodes and bedgeregions arrays.

Reply:
Isn't this done already automatically with lazy access, if not provided e.g. by TetGen ?
So we create simplexgrid as it is now and then trigger the bedge creation by 
e.g. `grid[BEdgeNodes]` ?

Reply:
If I understand you correctly, you day that we do not need an additional simplexgrid constructor that takes care of the boundary edges, etc.?

Reply:
I think we will need it, when coupling to tetgen or some other mesh generator capable of creating boundary edges. However, IMHO to be consistent, this then also should pass `bedges` before `bedgeregions`.